src/app/onboarding/components/create-new-archive/create-new-archive.component.scss

```diff
@@ -0,0 +1,130 @@
+@import '../welcome-screen/welcome-screen.component.scss';
+
+form {
+  margin: 0;
+  padding: 0;
+  max-width: 100%;
+}
+
+.two-buttons {
+  button {
+    @media only screen and (max-width: $mobile-breakpoint) {
+      width: 49%;
+    }
+    width: auto;
+    margin: 0;
+  }
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
+  margin: 2em 0;
+}
+
+.selected-archive {
+  & > div {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    max-width: 50%;
+  }
+
+  @media only screen and (max-width: $mobile-breakpoint) {
+    max-width: 100%;
+    border: 1px solid #C8C8C8;
+    box-sizing: border-box;
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+    border-radius: 8px;
+    padding: 20px;
+  }
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  max-width: 50%;
+  margin: 2em auto;
+}
+
+.name-field {
+  input {
+    max-width: 50%;
+  }
+  & > * {
+    margin: 10px;
+  }
+  display: flex;
+  flex-direction: row;
+  font-size: 1.5em;
+  justify-content: center;
+  align-items: center;
+}
+
+.archive-type-selector {
+  & > * {
+    @media only screen and (max-width: $mobile-breakpoint) {
+      width: 100%;
+    }
+    width: 30%;
+  }
+  @media only screen and (max-width: $mobile-breakpoint) {
+    flex-direction: column;
+  }
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: stretch;
+}
+
+.archive-type-name {
+  margin-bottom: 7px;
+  align-self: center;
```

Comment:
My Firefox dev tools say:

> `align-self` has no effect on this element since it’s not a grid or flex item.
>
> Try adding `display:grid`, `display:flex`, `display:inline-grid`, or `display:inline-flex` to the element’s parent. [Learn more](https://developer.mozilla.org/en-US/docs/Web/CSS/align-self)